graphql/concrete_notification/fields.ts

```diff
@@ -0,0 +1,13 @@
+import { Concrete_notification as ConcreteNotification, Notification } from "../generated";
+import { Authorized, FieldResolver, Resolver, Root } from "type-graphql";
+import { prisma } from "../../common/prisma";
+import { Role } from "../authorizations";
+
+@Resolver(of => ConcreteNotification)
+export class ExtendedFieldsConcreteNotificationResolver {
+    @FieldResolver(returns => Notification)
+    @Authorized(Role.UNAUTHENTICATED)
```

Comment:
Which privileges does this `UNAUTHENTICATED` role have? Why isn't this constrained to admins?

Reply:
Everyone has the Unauthenticated role, it basically disables authentication (explicitly). Cause everyone who can access a certain concrete_notification should be able to follow the edge to the corresponding notification. Also notifications are "public configuration" (description, interval, etc.) so there's no need to enforce strict authentication there.